### PR TITLE
feat(3218): hide empty offers table

### LINKED ISSUE #3218: hide offers tab on nft detail when no offers are made
remove this view:
image

? replace with message "There are no offers" ~

### PR DESCRIPTION
**Pull request by petersopko**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #3218
- [ ] Requires deployment <rubick/magick_issue>
- [x] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=<My_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address>)

### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Updated in https://github.com/kodadot/nft-gallery/pull/3231/commits/fc521fc4fe89aae83e27e379570e9ae3fc9d6dc2
image

### CODE REVIEW

File: components/bsx/Offer/OfferList.vue [hunk 1]
```diff
@@ -10,6 +10,9 @@
       :accountId="accountId"
       :isOwner="isOwner" />
   </CollapseCardWrapper>
+  <CollapseCardWrapper :label="$t('nft.offer.label')" v-else>
+    <p>{{ 'There are currently no offers for this NFT.' }}</p>
```
Comment (roiLeo) [inline components/bsx/Offer/OfferList.vue:14]: would it be possible to add this into en.json? key: `nft.offer.empty`

Comment (petersopko) [reply components/bsx/Offer/OfferList.vue]: added translation

File: langDir/en.json [hunk 2]
```diff
@@ -498,7 +498,8 @@
     },
     "offer": {
       "label": "Offers",
```
Comment (vikiival) [inline langDir/en.json:500]:
```suggestion
      "label": "Offers ({0})",
```

File: components/bsx/Offer/OfferList.vue [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <CollapseCardWrapper :label="$t('nft.offer.label')">
+  <CollapseCardWrapper :label="$t('nft.offer.label')" v-if="total">
```
Comment (vikiival) [inline components/bsx/Offer/OfferList.vue:2]:
```suggestion
  <CollapseCardWrapper :label="$t('nft.offer.label', [total])">
```

File: components/bsx/Offer/OfferList.vue [hunk 4]
```diff
@@ -10,6 +10,9 @@
       :accountId="accountId"
       :isOwner="isOwner" />
   </CollapseCardWrapper>
+  <CollapseCardWrapper :label="$t('nft.offer.label')" v-else>
+    <p>{{ $t('nft.offer.empty') }}</p>
+  </CollapseCardWrapper>
```
Comment (vikiival) [inline components/bsx/Offer/OfferList.vue:15]: This is handled by OfferTable

File: components/bsx/Offer/OfferList.vue [hunk 5]
```diff
@@ -117,3 +120,4 @@ export default class OfferList extends mixins(
   }
 }
 </script>
+``
```
Comment (vikiival) [inline components/bsx/Offer/OfferList.vue:123]:
```suggestion
```

**Review by vikiival — CHANGES_REQUESTED** [review]
Check suggestions

**Review by vikiival — APPROVED** [review]

**Pull request merged**